examples/src/index.js

```diff
@@ -40,6 +60,8 @@ const AsyncDemo = () => {
         onChalExpired={handleChallengeExpired}
       />
       <button onClick={executeCaptcha}>Execute asynchronously</button>
+      <button onClick={getRespKey}>Get Response Key asynchronously</button>
+      <button onClick={getResponse}>Get Response asynchronously</button>
```

Comment:
`Get Response Key` and `Get Response` return synchronously. We can remove `asynchronously` to avoid confusions.

Reply:
Done https://github.com/hCaptcha/react-hcaptcha/pull/122/commits/09a2cab4b5e9dae2f5543b4e5839100ad0618c5a